### PR TITLE
webui: allow deletion of dead poller nodes

### PR DESCRIPTION
**Pull request by murrant**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by alandeboer** [comment]
Worked for me but do I need to do anything about this validation warning?

> Your local git contains modified files, this could prevent automatic updates.
        [FIX] You can fix this with ./scripts/github-remove
        Modified Files:
         html/pages/pollers/pollers.inc.php

> 

I'm pretty new to this, sorry.

**Comment by murrant** [comment]
@doppiscantsleep Thanks for testing!

You can remove the modifications with `./scripts/github-remove` and wait for this to be merged.

File: html/includes/modal/delete_poller.inc.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            e.preventDefault();
+            var id = $("#id").val();
+            $.ajax({
+                type: 'POST',
+                url: 'ajax_form.php',
+                data: {type: "delete-poller", id: id},
+                success: function (result) {
+                    if (result.status == 0) {
+                        // Yay.
+                        $('#message').html('<div class="alert alert-info">' + result.message + '</div>');
```
Comment (laf) [inline html/includes/modal/delete_poller.inc.php]: Switch to use toastr? We've got a mix of message delivery at the moment and I prefer toastr - what do you think?

https://github.com/aldemira/librenms/blob/ce784ad59cd12d2c224bbebbb5aa8d119da6f65e/html/includes/modal/delete_host_dependency.inc.php#L58

**Comment by laf** [comment]
Just one comment from me.

**Comment by shimamizu** [comment]
I wanted to add this but you're already working on it, been having to remove old PoC pollers from MySQL haha, a webgui ability to remove them would be great!

**Pull request merged**